CHANGELOG.md

```diff
@@ -1,26 +1,25 @@
 ## 2.0.0-beta1
-
 March 29th, 2018
```

Comment:
This actually went out April 2nd, or 3rd?

Reply:
I can change it if it matters enough.  The *.nupkg was submitted to IT for A/V scan on March 30.